### DISCUSSION TITLE
UI and functionality questions

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [ahrm/sioyek#659] [Q&A]
Hi! am a new user that just migrated from `zathura` and have a few questions :
Is there a way to style/move the command menu ? like center it instead of it starting from the top?
Also i struggled a bit with the default behavior of using the same instance of `sioyek` for opening files, i found this 
```
# (deprecated, use `should_launch_new_window` instead) If it is 0, then we use the previous instance of sioyek when launching a new file.
# otherwise a new instance is launched every time we open a new file.
should_launch_new_instance				1

# If set, we open a new sioyek window when a new file is opened, otherwise we open the file in the previous window
should_launch_new_window				1

```
Why is `should_launch_new_instance` deprecated , `should_launch_new_window` doesn't really replace it, since if i open 10 windows and close one they all die.

what is the intended way of using this ? Maybe using the CLI options `--new-window ,--reuse-window  ` ?

**Comment by ahrm** [comment]
> Is there a way to style/move the command menu ? like center it instead of it starting from the top?

No.

> since if i open 10 windows and close one they all die

No they don't (unless you close them using the `quit` command instead of `close_window` command).

> what is the intended way of using this ? Maybe using the CLI options --new-window ,--reuse-window   ?

Use `--new-widow` if you want to open a new window. You can set it as the default in the preferences in which case you can use `reuse-window` to reuse the previous window.
